automation/taskcluster/decision_task.py

```diff
@@ -30,7 +30,7 @@
 BUILDER = TaskBuilder(
     task_id=os.environ.get('TASK_ID'),
     repo_url=REPO_URL,
-    branch=os.environ.get('MOBILE_HEAD_BRANCH'),
+    git_ref=os.environ.get('MOBILE_HEAD_BRANCH'),
```

Comment:
Can we rename `MOBILE_HEAD_BRANCH` to `MOBILE_HEAD_REF` instead?

Reply:
I wouldn't do it now. This variable is prefixed by `MOBILE_` which means, it's used by Chain of Trust. We'd need to deploy a new version.